universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockPredicatesEditView.java

```diff
@@ -266,9 +266,8 @@ private ContainerTag createActionDropdown(String blockName) {
 
   private Tag createHiddenQuestionDefinitionInput(QuestionDefinition questionDefinition) {
     return input()
-        .isHidden()
         .withName("questionId")
-        .withType("number")
+        .withType("hidden")
```

Comment:
question (non-blocking): I'm not familiar with this, what does a `hidden` type mean? Is it the same as a `hidden` class?